libraries/CAN/src/CANXMC.cpp

```diff
@@ -33,9 +42,14 @@ namespace ifx
   }
   CANXMC::~CANXMC() {}
 
+  int CANXMC::setDeviceId(long id = 0x12) { // TODO: delete in the future!
```

Comment:
This id should not have a default value I guess?

Reply:
fix